src/app/pages/apps/components/available-apps/available-apps.component.ts

```diff
@@ -66,16 +81,65 @@ export class AvailableAppsComponent implements OnInit, AfterViewInit {
     return `${app.catalog.id}-${app.catalog.train}-${app.name}`;
   }
 
-  private loadTestData(): void {
-    // TODO: Temporary
+  trackByAppSectionTitle(_: number, appSection: AppSection): string {
+    return `${appSection.title}`;
+  }
+
+  private loadApplications(): void {
     this.loader.open();
-    this.appService.getAllCatalogs().pipe(
-      catalogToAppsTransform(),
-      untilDestroyed(this),
-    ).subscribe((apps) => {
-      this.apps = apps;
-      this.loader.close();
-      this.cdr.markForCheck();
+
+    combineLatest([
+      this.appService.getAllCatalogs().pipe(
+        catalogToAppsTransform(),
+      ),
+      this.appService.getAllAppCategories(),
+    ])
+      .pipe(untilDestroyed(this))
+      .subscribe(([apps, appCategories]) => {
+        this.setupApps(apps, appCategories);
+        this.loader.close();
+        this.cdr.markForCheck();
+      });
+  }
+
+  private setupApps(apps: CatalogApp[], appCategories: string[]): void {
+    this.apps = apps;
+
+    this.allRecommendedApps = this.apps.filter((app) => app.recommended);
+    this.allNewAndUpdatedApps = this.apps
```

Comment:
Wouldn't this just sort all apps in an order. Meaning all apps are technically "new and updated". Just further down the list. We should have a threshold. Like maybe apps last updated this week or this month. Or apps with versions newer than what the user has installed or something similar. Figuring this out maybe more difficult. We can discuss and create a new ticket if it deviates too much from the scope of this one.

Reply:
maybe we should show latest updated apps with 6 in limit with no possibility to show more?

I think if we want to improve it we can do it in terms of new ticket.